app/views/images/_index.html.erb

```diff
@@ -1,10 +1,28 @@
-<h4>Images</h4>
+<div>
+<h4 style="display: inline-block;">Images</h4>
+<div style="display: inline-block; float: right; margin-right: 32px; margin-bottom: 10px;" class="image-order">
```

Comment:
Can we pull this inline styling out into a separate css file? I think it'll be good to create a file for the sidebar! And also for the inline styling above it because at that point might as well